numpy/polynomial/_polybase.py

```diff
@@ -887,7 +887,7 @@ def roots(self):
         """Return the roots of the series polynomial.
 
         Compute the roots for the series. Note that the accuracy of the
-        roots decrease the further outside the domain they lie.
+        roots decreases the further outside the `domain` they lie.
```

Comment:
I don't think that it's actually grammatically wrong to use "decrease" instead of "decreases". I'm also not sure if you're supposed to use backticks when writing out method names in documentation. I know that at least some classes don't do that in their documentation.

Reply:
@Unessam Thank you for submitting your PR!
@MatteoRaso Thank you for taking the time to review it!
Indeed, backticks are not required in this instance since the word “domain” is referring to the domain attribute in the numpy.polynomial package (https://numpy.org/doc/stable/reference/generated/numpy.polynomial.polynomial.Polynomial.domain.html#numpy.polynomial.polynomial.Polynomial.domain). 
As far as your second comment, since the verb `decrease` refers to the noun `accuracy`, the proposed change by @Unessam is grammatically correct. `Tends to decrease` would be a more accurate description in this case though.